drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -103,7 +102,7 @@ static int _send(netdev_t *netdev, const struct iovec *vector, unsigned count)
     at86rf2xx_tx_prepare(dev);
 
     /* load packet data into FIFO */
-    for (unsigned i = 0; i < count; i++, ptr++) {
+    for (unsigned i = 0; i < count; ++i, ++ptr) {
```

Comment:
This has the same effect, but okay :P

Reply:
Haha it seems faster though ... brains are stupid :P